apps/air-discount-scheme/README.md

```diff
@@ -0,0 +1,81 @@
+# Air Discount Scheme - Loftbru
+
+## About
+
+Generates discount codes that can be used for booking domestic flights online.
+
+There are certain precondition to be eligible for a discount. They are:
+
+- The person's legal domicile needs to be in a predefined set of towns outside
+  the capital (We fetch postal codes from Þjóðskrá to validate if user is
+  eligible).
+
+## Project entrypoints
+
+Dev: [https://loftbru.dev01.devland.is](https://loftbru.dev01.devland.is)
+Staging: [https://loftbru.staging01.devland.is](https://loftbru.staging01.devland.is)
+Production: [https://loftbru.island.is](https://loftbru.island.is)
```

Comment:
![Screenshot 2020-09-14 at 11 13 18](https://user-images.githubusercontent.com/5694851/93079232-54a91980-f67b-11ea-924e-d114db3f8fa6.png)

This will look like this. Can add `extra line`/`<br />`/`ul`